.travis.yml

```diff
@@ -1,14 +1,24 @@
 language: python
+os:
+    - linux
+    - osx
 python:
     - 2.7
     - 3.4
+env:
+    - JULIA_VERSION = "LAST_RELEASE"
+    - JULIA_VERSION = "NIGHTLY"
 compiler:
   - clang
 notifications:
   email: false
 before_install:
   - sudo add-apt-repository ppa:staticfloat/julia-deps -y
-  - sudo add-apt-repository ppa:staticfloat/juliareleases -y
+  if [ "$JULIA_VERSION" = "LAST_RELEASE" ]; then
+      - sudo add-apt-repository ppa:staticfloat/juliareleases -y
+  else
+      - sudo add-apt-repository ppa:staticfloat/juliareleases -y
+  end 
```

Comment:
could modernize https://github.com/JuliaLang/julialang.github.com/blob/master/install-julia.sh (and take out the ppa parts) which approximates what `language: julia` does on travis. Travis doesn't support multiple language settings afaik, and `language: python` is probably more complicated.